components/UserSummary.tsx

```diff
@@ -113,10 +118,11 @@ export default function UserSummary({
           titleIsLink && user ? (
             <StyledLink
               href={routeToUser(user.username)}
+              target="_blank"
```

Comment:
```suggestion
              target="_blank"
              rel="noopener noreferrer"
```
Mostly for security reasons in older browsers!